src/effects/attack_result_text.c

```diff
@@ -171,35 +171,16 @@ void func_E0090428(EffectInstance* effect) {
     func_E0090444(effect);
 }
 
-// gSPTextureRectangle needs a lot of work
-#ifdef NON_EQUIVALENT
 void func_E0090444(EffectInstance* effect) {
     Matrix4f sp18;
     Matrix4f sp58;
     AttackResultTextFXData* data;
     s32 unk_00;
     s32 unk_1C;
-    s32 temp_a2;
-    s8* temp_a0_3;
+    s32 z;
     f32 scale;
-    // f32 temp_f2;
-    // s32 temp_f4;
-    // s32 temp_f4_2;
-    // s32 temp_f4_3;
-    // s32 temp_f4_4;
-    // s32 temp_f4_5;
-    // s32 temp_v0_3;
-    // s32 temp_v1_4;
-    // u32 phi_v1;
-    // u32 phi_a0;
-    // u32 phi_v1_2;
-    // u32 phi_v0;
-    // u32 phi_v0_2;
-    // s32 phi_a0_2;
-    // s32 phi_v1_3;
-    // s32 phi_v0_3;
-
-    data = effect->data;
+
+    data = effect->data.attackResultText;
```

Comment:
Could you move the data temp to the top of the list and initialize it on that same line? same with unk_00 and unk_1C following. sp18 and sp58 should be okay after those 3